src/test/ui/binop/binop-consume-args.stderr

```diff
@@ -252,10 +292,15 @@ error[E0382]: use of moved value: `lhs`
 LL | fn shl<A: Shl<B, Output=()>, B>(lhs: A, rhs: B) {
    |                                 --- move occurs because `lhs` has type `A`, which does not implement the `Copy` trait
 LL |     lhs << rhs;
-   |     --- value moved here
+   |     ---------- `lhs` moved due to usage in operator
 LL |     drop(lhs);
    |          ^^^ value used here after move
    |
+note: calling this operator moves the left-hand side
+  --> $SRC_DIR/libcore/ops/bit.rs:LL:COL
```

Comment:
Are there other errors where we include snippets from libcore/libstd? I can see this both being helpful and confusing - seeing a `fn(self)` could help make why this error is happening more concrete, but simultaneously it might be unexpected to see code that isn't your own in the errors. Not sure..

Reply:
There are many such cases: for example, https://github.com/rust-lang/rust/search?p=4&q=src_dir&unscoped_q=src_dir